kotlin/usecases/subpub_app_android/src/main/kotlin/com/example/androidsubpub/MainActivity,kt

```diff
@@ -0,0 +1,276 @@
+/*
```

Comment:
The name of this file has ',kt' (that's 'comma kt') extension, shouldn't it be '.kt' ('period kt')?

Reply:
fixed